data/korath outfits.txt

```diff
@@ -91,6 +91,15 @@ outfit "Large Heat Shunt"
 	"cooling" 54
 	description "Without these powerful cooling systems, most Korath warships would overheat within seconds as soon as they began to fire their weapons."
 
+outfit "Fuel Collector"
+	category "Systems"
+	cost "101000"
```

Comment:
um.

Reply:
It is as effective as two fuel pods and one emergency ramscoop, which cost 2*20000+72000=114000 together; and because it's less flexible, it probably ought to cost somewhat less, hence the arbitrary value of 101000.

Reply:
Perhaps too arbitrary.

Reply:
That's not the problem here...

(There are quotation marks on the price.)

Reply:
That ought to be fine.